cmd/utils.go

```diff
@@ -25,7 +25,6 @@ const (
 	paymentFlagName      = "paymentAddress"
 	secondarySPFlagName  = "secondarySPs"
 	contentTypeFlagName  = "contentType"
-	txnHashFlagName      = "txnHash"
 	startOffsetFlagName  = "start"
```

Comment:
wondering if we can rename to` Flag.....` so that can easily to know it is flag..

Reply:
a litte strange to begin with Flag , change it like txnHashFlag.